codeclimate-govet.go

```diff
@@ -1,30 +1,45 @@
 package main
 
-import "github.com/codeclimate/cc-engine-go/engine"
-import "strings"
-import "os"
-import "os/exec"
-import "strconv"
+import (
+	"github.com/codeclimate/cc-engine-go/engine"
+	"strings"
+	"os"
+	"os/exec"
+	"strconv"
+	"fmt"
+)
 
 func main() {
 	rootPath := "/code/"
 
 	config, err := engine.LoadConfig()
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error loading config: %v\n", err)
 		os.Exit(1)
 	}
 
 	analysisFiles, err := engine.GoFileWalk(rootPath, engine.IncludePaths(rootPath, config))
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error initializing: %v\n", err)
```

Comment:
I _think_ they look the same. 

For these errors at least, they both look like: 
```
Error: exit status 1
Gofmt output: /code/src/meta.go:23: Value passes lock by value: main.Meta
/code/src/meta.go:24: return copies lock value: main.Meta
/code/src/meta.go:27: Print passes lock by value: main.Meta
/code/src/meta.go:37: Refl passes lock by value: main.Meta

Error analyzing path: /code/src/stack.go
Error: exit status 1
Gofmt output: /code/src/stack.go:38: Value passes lock by value: main.Stack
/code/src/stack.go:39: return copies lock value: main.Stack
/code/src/stack.go:42: Refl passes lock by value: main.Stack
/code/src/stack.go:73: Print passes lock by value: main.Stack
```